cmd/operator-sdk/new/cmd.go

```diff
@@ -59,7 +58,7 @@ generates a skeletal app-operator application in $HOME/projects/example.com/app-
 	newCmd.Flags().StringVar(&operatorType, "type", "go", "Type of operator to initialize (choices: \"go\", \"ansible\" or \"helm\")")
 	newCmd.Flags().StringVar(&depManager, "dep-manager", "modules", `Dependency manager the new project will use (choices: "dep", "modules")`)
 	newCmd.Flags().StringVar(&repo, "repo", "", "Project repository path for Go operators. Used as the project's Go import path. This must be set if outside of $GOPATH/src with Go modules, and cannot be set if --dep-manager=dep")
-	newCmd.Flags().BoolVar(&skipGit, "skip-git-init", false, "Do not init the directory as a git repository")
+	newCmd.Flags().BoolVar(&gitInit, "git-init", false, "Initialize the project directory as a git repository")
```

Comment:
```suggestion
	newCmd.Flags().BoolVar(&gitInit, "git-init", false, "Initialize the project directory as a git repository ("false" by default)")
```

Is it worth mentioning that its false by default wdyt?

Reply:
Yes I'll mention that since it's in the CLI reference as well.